shared_requirements.txt

```diff
@@ -579,3 +579,17 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-core-experimental azure-core<2.0.0,>=1.25.0
 #override azure-developer-devcenter isodate<1.0.0,>=0.6.1
 #override azure-developer-devcenter azure-core<2.0.0,>=1.24.0
+
+# azure-ai-ml
+pyyaml<7.0.0,>=5.1.0
+marshmallow<4.0.0,>=3.5
+jsonschema<5.0.0,>=4.0.0
+tqdm<=5.0.0
+strictyaml<2.0.0
+colorama<=0.5.0
```

Comment:
I think this was `colorama<0.5.0`.... this one is the most annoying, and it _probably_ should be `colorama>=4.0`, but because it's not a stable version it's hard to know how stable the package is...